rero_ils/modules/ill_requests/utils.py

```diff
@@ -29,3 +29,16 @@ def get_pickup_location_options():
             location = Location.get_record_by_pid(pid)
             location_name = location.get('pickup_name', location.get('name'))
             yield (location.pid, location_name)
+
+
+def get_document_identifiers(doc, types=None):
+    """Get document identifiers.
+
+    :params: doc: document
+    :params: types: available types of identifiers
```

Comment:
cosmetic : `:param type: list of identifier type to retrieve.`
this is not `:params: type:` but just `:param type:` ;-)
But it's correct for `:returns:` XD Yessss, I know you like python !